.github/workflows/ios_tag_release.yml

```diff
@@ -94,6 +98,12 @@ jobs:
           *) echo "👎 Not a release or hotfix branch"; exit 1 ;;
         esac
 
+    - name: Validate commit SHA
+      if: inputs.commit-sha
+      run: |
+        git merge-base --is-ancestor "${{ inputs.commit-sha }}" "origin/${{ env.BRANCH }}" \
+          || (echo "👎 Commit ${{ inputs.commit-sha }} does not belong to ${{ env.BRANCH }} branch" && exit 1)
```

Comment:
This step verifies that the commit belongs to the release branch – this shouldn’t ever fail because the commit SHA is passed here by the automation, not by the human, and the automation is making that commit on the release branch. Yet I’ve seen it failing because of an unrecognized branch, and I had to add `origin/` prefix to fix the branch ref.

Reply:
I’ve been considering it and decided to keep it in the workflow, but forgot the exact reason. Thinking about it now it should be fine to keep it in the plugin. I’ve added a comment to tackle it separately and will also add a task in Asana later. Thanks!